### PR TITLE
more build fixes

### PR DESCRIPTION
**Pull request by nmartin413**
### Background

Linter wasn't running. This gets things back to spec

### Changes

* update type annotations

### Testing

* `make lint test`

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,5 @@
 packages = panther_core
 
-ci: lint test
```
Comment (darwayne) [inline Makefile:3]: was this removed on purpose?

**Review by darwayne — COMMENTED** [review]
Looks like ci isn’t happy

File: Makefile (hunk 1 above)
Comment (nmartin413) [reply Makefile:3]: yes - i add two explicit steps to the circle yaml

File: setup.py [hunk 2]
```diff
@@ -16,13 +16,13 @@
...
     package_dir={'exec': 'panther_core/exec'},
-    version='0.0.11',
+    version='0.1.0',
     license='AGPL-3.0',
     description='Panther core library',
     author='Panther Labs Inc',
     author_email='pypi@runpanther.io',
     url='https://github.com/panther-labs/panther_core',
-    download_url='https://github.com/panther-labs/panther_core/archive/v0.0.11.tar.gz',
+    download_url='https://github.com/panther-labs/panther_core/archive/v0.1.0.tar.gz',
```
Comment (nmartin413) [inline setup.py:25]: marking to `0.1.0` because of the changes to the `ExecutionMatch` class field names.

File: panther_core/exec/common.py [hunk 3]
```diff
@@ -16,15 +16,15 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 
+from __future__ import annotations
```
Comment (nmartin413) [inline panther_core/exec/common.py:19]: @lindsey-w @cdzombak from what i've been able to grok, this is fine if you can predict the python versions you'll run under?

**Comment by nmartin413** [comment]
> Looks like ci isn’t happy

@darwayne ci happified ++

**Review by darwayne — APPROVED** [review]

**Pull request merged**